src/plugins/data/public/index.ts

```diff
@@ -45,12 +45,15 @@ import {
   UrlFormat,
   StringFormat,
   TruncateFormat,
+  serializeFieldFormat,
```

Comment:
Considering its in a namespace, maybe import `serializeFieldFormat as serialize`?
Also, why is the serialize a utility, but the deserialize is a method on the public contract of the plugin :thinking: ?

Reply:
as deserialize needs access to the registry